src/dlgColorTrigger.cpp

```diff
@@ -37,104 +37,104 @@ dlgColorTrigger::dlgColorTrigger(QWidget* pF, TTrigger* pT, int mode) : QDialog(
 {
     // init generated dialog
     setupUi(this);
-    connect(closeButton, SIGNAL(pressed()), this, SLOT(slot_save_and_exit()));
-    connect(pushButton_black, SIGNAL(clicked()), this, SLOT(setColorBlack()));
+    connect(closeButton, &QAbstractButton::pressed, this, &dlgColorTrigger::slot_save_and_exit);
+    connect(pushButton_black, &QAbstractButton::clicked, this, &dlgColorTrigger::setColorBlack);
     QPalette palette;
     QString styleSheet;
     QColor color;
     color = mpTrigger->mpHost->mBlack;
     palette.setColor(QPalette::Button, color);
-    styleSheet = QString("QPushButton{background-color:") + color.name() + QString(";}");
+    styleSheet = QStringLiteral("QPushButton{background-color:") + color.name() + QStringLiteral(";}");
```

Comment:
I've got so feed up with this less than optimum construct (using `QString::operator+`) I've abstracted it out to a method that will also set a contrasting fore-ground colour (`Qt::white` or `Qt::black`) in one of my pending things, something like:

    styleSheet = QStringLiteral("QPushButton {color: %1; background-color: %2; }").arg((color.lightness() > 127) ? QStringLiteral("black") : QStringLiteral("white"), color.name());

Reply:
... but that is a whole *different* PR so I'll let this change stand I suppose.